src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -358,8 +358,10 @@ protected string GeneratePropertyName(string fieldName)
                 baseName = fieldName;
             }
 
-            // Make uppercase the first letter
-            return char.ToUpper(baseName[0]).ToString() + baseName.Substring(1);
+            // Make the first character upper case using the "en-US" culture.  See discussion at
+            // https://github.com/dotnet/roslyn/issues/5524.
+            var firstCharacter = CompletionRules.EnUSCultureInfo.TextInfo.ToUpper(baseName[0]);
```

Comment:
I considered adding `Features\Core\Portable\Common\CultureSomethingHelper.cs` but ultimately decided against it due to that class only containing one item, but I could be very easily convinced otherwise.